shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -903,6 +903,12 @@ - (UITextContentType)textContentType {
 // Prevent UIKit from showing selection handles or highlights. This is needed
 // because Scribble interactions require the view to have it's actual frame on
 // the screen.
+//
+// These are not documened methods. On iOS 17, the insertion point color is also
+// used to paint the highlighted background of the selected IME candidate:
```

Comment:
what else do we use this function for?

Reply:
The paragraph above says that it's used to make sure the system caret / selection handles aren't visible.